src/quo2/components/common/unread_grey_dot/view.cljs

```diff
@@ -5,4 +5,5 @@
 (defn unread-grey-dot
   []
   [rn/view
-   {:style style/unread-grey-dot}])
+   {:style               style/unread-grey-dot
```

Comment:
just incase you're not aware of this, sometimes you have to add `:accessible true` for the accessibility label to work on `rn/view`'s